test/integration/h2_capture_fuzz.proto

```diff
@@ -132,6 +132,11 @@ message H2FrameMetadata {
   Metadata metadata = 3;
 }
 
+message H2FrameStatus {
```

Comment:
Can you please add a comment what this frame is?
(IIUC it's a response with a single status header frame).

Reply:
Not necessarily a response, can also be a "bad request" with a status header?